apps/api/v2/src/modules/organizations/attributes/options/organizations-attributes-options.controller.ts

```diff
@@ -118,6 +131,30 @@ export class OrganizationsAttributesOptionsController {
     };
   }
 
+  @Roles("ORG_MEMBER")
+  @PlatformPlan("ESSENTIALS")
+  @Get("/attributes/:attributeId/options/assigned")
+  @ApiOperation({ summary: "Get all attribute options that are assigned to users" })
+  async getOrganizationAttributeAssignedOptions(
+    @Param("orgId", ParseIntPipe) orgId: number,
+    @Param("attributeId") attributeId: string,
+    @Query() queryParams: GetAssignedAttributeOptions
+  ): Promise<GetAllAttributeAssignedOptionOutput> {
+    const { skip, take, ...rest } = queryParams;
+    const attributeOptions =
+      await this.organizationsAttributesOptionsService.getOrganizationAttributeAssignedOptions(
+        orgId,
+        attributeId,
+        skip,
+        take,
+        rest
+      );
+    return {
+      status: SUCCESS_STATUS,
+      data: attributeOptions,
+    };
```

Comment:
This endpoint seems to be added at organization level and not on organization's team level.

So, this works  - {{API_URL}}/v2/organizations/5/attributes/ce3d5bbc-bd7b-4bbf-9c43-c892ae524b41/options/assigned
![image](https://github.com/user-attachments/assets/bf116f39-275d-4c2e-8d0d-e5be0982e162)

but not this
![image](https://github.com/user-attachments/assets/5ce29513-eb44-4465-b23c-1dbbb721f27f)

Reply:
Also, using attributeSlug doesn't work

i.e.  {{API_URL}}/v2/organizations/5/attributes/`service`/options/assigned doesn't work